toolchain/gcc/PKGBUILD

```diff
@@ -121,11 +121,17 @@ build() {
   export CXXLAGS+=" -fno-semantic-interposition"
 
   if [ -n "$_optimize_v3" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
-  elif [-n "$_optimize_v4" ]; then
-    export ASFLAGS="-D__AVX__=1 -D__AVX2__=1 -D__AVX512__=1 -msse2avx -D__FMA__=1 "
+    export ASFLAGS+="-D__AVX__=1 -D__AVX2__=1 -msse2avx -D__FMA__=1 "
+    export LDFLAGS+=" -Wl,-z,max-page-size=0x1000"
+    export CFLAGS+=" -Wl,-z,max-page-size=0x1000"
+    export CXXFLAGS+=" -Wl,-z,max-page-size=0x1000"
```

Comment:
Btw, `max-page-size` parameter is only for linker, there is no sense to specify it in CXFLAGS/CXXFLAGS.